lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -841,12 +836,33 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
             raw_pointer_type);
       }
       return m_address_sp;
-    case 1:
+    case 1: {
+      // TypeMangling for "Swift.UInt64"
+      CompilerType uint64_type =
+          m_ts->GetTypeFromMangledTypename(ConstString("$ss6UInt64VD"));
       RETURN_CHILD(m_id_sp, id, uint64_type);
-    case 2:
-      RETURN_CHILD(m_kind_sp, kind, uint32_type);
-    case 3:
+    }
+    case 2: {
+      // TypeMangling for "Swift.TaskPriority"
+      CompilerType priority_type =
+          m_ts->GetTypeFromMangledTypename(ConstString("$sScPD"));
       RETURN_CHILD(m_enqueue_priority_sp, enqueuePriority, priority_type);
+    }
+    case 3: {
+      if (!m_child_tasks_sp) {
+        const auto &tasks = m_task_info.childTasks;
+        std::string mangled_typename =
+            mangledTypenameForTasksTuple(tasks.size());
+        CompilerType tasks_tuple_type =
+            m_ts->GetTypeFromMangledTypename(ConstString(mangled_typename));
+        DataExtractor data{tasks.data(), tasks.size() * sizeof(tasks[0]),
```

Comment:
you're not really dereferencing it, but the access to tasks[0] without checking size() makes me uneasy. Why not just use the type name here?

Reply:
I used `tasks[0]` to avoid hard coding the type – which would leave the potential of skew in future changes (if for example the element of `tasks` were changed.

I'll refactor this to another way to achieve the same.